vars/pmmVersion.groovy

```diff
@@ -25,6 +26,8 @@ def call(type='latest') {
       return versions
     case 'list':
       return versionsList
+    case 'ovf':
+      return ovfVersions
```

Comment:
Maybe we can return the latest 5-7 versions? Like this:
```
versionsList.reverse().subList(0, 5)
```

Reply:
I have spent some time trying to make it work, but it seems it keeps failing, for example see here, I tried to reverse the versionList and return subList, Jenkins started complaining, https://pmm.cd.percona.com/job/pmm2-ovf-upgrade-tests-matrix/21/console I would fix this later, for now please lets use dedicated versionList of OVF on the same file